### PR TITLE
get_ride_screen

### PR DESCRIPTION
**Pull request by pranayshl**

### CODE REVIEW

File: lib/main.dart
Comment (maheshalyana) [inline lib/main.dart:1]: don't make changes in `main.dart`

File: lib/screens/get_ride_screen.dart [hunk 1]
```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/material.dart';
+
+class Ridebhaiya extends StatefulWidget {
```
Comment (maheshalyana) [inline lib/screens/get_ride_screen.dart:3]: change this class name to  screen name i.e. `GetRideScreen`

File: pubspec.yaml [hunk 2]
```diff
@@ -64,8 +66,8 @@ flutter:
 
   # To add assets to your application, add an assets section, like this:
   assets:
-    - assets/
-
+    - assets/ridebhi.png
```
Comment (maheshalyana) [inline pubspec.yaml:69]: just add images in assets folder, you don't need to mention the image here, just mentioning the assets folder here is enough, revert this change

**Review by maheshalyana — CHANGES_REQUESTED** [review]

**Review by Abhi-0987 — COMMENTED** [review]
image
image

Screen matches with the design and the test cases passed. Moving it to acceptance.
